frontend/src/environments/environment.ts

```diff
@@ -4,7 +4,9 @@
 
 export const environment = {
   production: false,
-  backendApiUrl: 'http://host.docker.internal:8080/',
+  backendApiUrl: 'http://host.docker.internal:8080',
+  backendNotificationUrl: 'localhost',
+  backendNotificationPort: 61614,
```

Comment:
These environment variables are also defined in the docker-compose file. Do we have to define them twice?

Reply:
Yes. When docker is building a productive build, the environment.ts is replaced by the environment.prod.ts and so the docker envoroment variables are used. While developing, we start the frontend using "ng serve" and do not make a productive docker build every time. Therefore, the variables are twice.